tests/test_rpe.py

```diff
@@ -0,0 +1,14 @@
+#
+#  Copyright (C) 2024 RapidSilicon
+#  Authorized use only
+#
+import pytest
+# from backend.rpe import RPE
+
+
+def test_rpe_1():
+    assert 1 == 1
+    # rpe = RPE('/home/guanyung/dev/rpe-poc/python/etc/device.xml')
```

Comment:
Please add TODOs in code instead of commenting code when possible.

Reply:
fixed